recipes/python-can/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "python-can" %}
+{% set version = "2.1.0" %}
+{% set sha256 = "4a5c01dd67feeda35f88e6c12ea14ac8cabd426b9be0cc5f9fd083fe90a9dbfc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - pyserial
+    - setuptools
+
+test:
+  imports:
+    - setuptools
```

Comment:
Why are you testing if you can import nose?

Reply:
The imports section is just testing if conda can import the library, so this should just be your library.